assets/src/edit-story/components/canvas/editLayer.js

```diff
@@ -43,13 +49,19 @@ const EditPageArea = withOverlay(styled(PageArea).attrs({
 `);
 
 function EditLayer({}) {
+  const ref = useRef(null);
   const {
     state: { currentPage },
   } = useStory();
   const {
     state: { editingElement: editingElementId },
+    actions: { clearEditing },
   } = useCanvas();
 
+  useKeyDownEffect(ref, { key: 'esc', editable: true }, () => {
+    clearEditing();
+  });
```

Comment:
```js
useKeyDownEffect(ref, { key: 'esc', editable: true }, () => clearEditing());
```

Reply:
Hmm. I thought prettier would do this for us. Fixed.